docs/download/data-access.md

```diff
@@ -3,18 +3,31 @@
 
 ## Data Access Control Bodies
 
-ICGC projects have two separate access control bodies. US based projects are controlled by dbGap, while non-US projects are authorized by the ICGC Data Access Compliance Office (DACO). To download controlled data, user must apply for access at corresponding data access control body.
+ICGC projects have two separate access control bodies. US based projects are authorized by dbGaP, while non-US projects are authorized by the ICGC Data Access Compliance Office (DACO). To download controlled data, a user must apply for access at the corresponding data access control body. The following table shows you which repository you are able to download from if you have been granted authorization from dbGaP or ICGC DACO.
 
-### ICGC DACO
 
-Follow the proceedure outlined at the [DACO](https://icgc.org/daco) page.
+| *Authorization From:*  | Data Portal | Collaboratory |  AWS |  EGA  | GDC   |  PDC  |
+| :--------------------| ----------- | ------------- | ---- | ----- | ----- | ----- |
+|   *ICGC DACO*	       |  ![checkmark](images/check_mark.png)  | ![checkmark](images/check_mark.png) | ![checkmark](images/check_mark.png) | ![checkmark](images/check_mark.png) |       |       |
+|   *dbGaP/eRA Commons*  |             |               |      |       | ![checkmark](images/check_mark.png)  | ![checkmark](images/check_mark.png) |
+
+
+### Obtaining Access to ICGC-Controlled Data in Data Portal, Collaboratory, AWS and/or EGA 
+
+1. Create an ICGC account at [https://icgc.org/user/register](https://icgc.org/user/register)
+2. Follow the procedure of applying for DACO access at [DACO](https://icgc.org/daco). **NOTE: Please ensure you submit your Gmail email address where it asks for an "OpenID"**
+3. Submit your DACO application
+4. DACO will notify you when your DACO application has been approved, or if they require any further information
```

Comment:
Should we also list how long the approval process usually takes?

Reply:
updated to include time estimate